_component_design/buttons-bar.md

```diff
@@ -0,0 +1,176 @@
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use a button bar in the following situations:
+
+- When there is a primary action represented by a simple button and you want to list additional actions in a drop-down menu.
+- To offer a number of action buttons at once and conserve horizontal space.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+In addition to the [Simple Buttons]({{site.baseurl}}/components/simple-buttons.html) best practices,
+consider the following best practices when you construct a button bar:
+
+- Do not use a button bar for objects with a small number of actions. For example, do not use a button bar for table objects that have only a create and delete action. When a table object has a small number of actions, use table controls at the top of the table. For more information, see [tables]({{site.baseurl}}/components/tables.html).
+- Do not mix button weights or sizes in a button bar.
+- Place the drop-down button as the last button in the bar.
+- Destructive actions should be placed inside the drop-down menu, and not as a visible button in the bar.
+- Do not use a tertiary button in a button bar.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Specifications
+
+Use the following specifications when you construct a button bar.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Composition
+
+A button bar contains the following elements:
+
+- **Simple button (required)**: The simple button represents the primary action that the user can take. A button bar can contain multiple simple buttons.
+- **Drop-down button (optional)**: When the user clicks the drop-down button, additional actions appear on a drop-down menu. Include only one drop-down button in a button bar.
+- **Divider (required)**: The divider separates the buttons in a button bar.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Button bar composition"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/inputs-and-controls/buttons-bar/buttonbar-composition.png" width="284"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Style
+
+Use these specifications when you construct a button bar.
```

Comment:
Use the following specifications when you construct a button bar:

Reply:
My convention is to use 'these' instead of 'following' because the specs don't actually follow the text. The specs are in a visual to the right of the text.

Reply:
Ah, gotcha!